src/features/select-book/selectBookSlice.ts

```diff
@@ -65,7 +65,18 @@ export const {
 } = selectBookSlice.actions;
 
 export const selectAvailableBooks = (state: RootState) => {
-  return state.content.books;
+  //define and filter array of viewable books
+  let ViewableBooks: Array<IBook> = state.content.books.filter( book => 
+    ( 
+      (book.fields.type === "pre-test" || "post-test") 
```

Comment:
I think this logic actually means something slightly different from what you intended.

You unfortunately have to be pretty wordy and say `book.fields.type === "pre-test" || book.fields.type === "post-test"`. But if you do the earlier suggestion to create a `Book.isAssessment` function you could replace this bit of logic with that!